kolibri/core/content/management/commands/importcontent.py

```diff
@@ -330,37 +331,45 @@ def _transfer(  # noqa: max-complexity=16
                     filetransfer = transfer.FileCopy(
                         srcpath, dest, cancel_check=self.is_cancelled
                     )
+                    file_transfers.append((f, filetransfer))
 
-                try:
-                    status = self._start_file_transfer(
+            with concurrent.futures.ThreadPoolExecutor(max_workers=5) as executor:
+                future_file_transfers = {}
+                for f, filetransfer in file_transfers:
+                    future = executor.submit(self._start_file_transfer,
```

Comment:
I am unsure if this will maintain progress tracking, as we currently set a threadlocal variable to allow access to the current task object (like `celery.current_task` in Celery and `get_current_job` in rq), so inside the context of the child thread here, that might be undefined. It won't error, but the progress per file will probably not be updated.

Reply:
Okay, thanks for the heads up! I'll take a look at this. I did notice that when running the management command, I was randomly seeing per file progress bars appear and disappear, and that stopped after adding the threads.

Reply:
We might want to just disable the per download progress tracking, because it becomes less meaningful when we have multiple concurrent downloads?

Reply:
Done, thanks!